<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopupBusinessActionWidget</class>
 <widget class="QWidget" name="PopupBusinessActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="usersLabel">
       <property name="text">
        <string>for</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectUsersButton">
       <property name="text">
        <string notr="true">&lt;All users&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>99</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="settingsButton">
       <property name="text">
        <string>Global Notification Settings...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QCheckBox" name="forceAcknoledgementCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Force Acknowledgment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="nx::vms::client::desktop::HintButton" name="hintLabel" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::HintButton</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/hint_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
